docs-src/app-dev-context/ide-debug.md

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
The document should mention this applies to workers _and_ replayers (even though it's kind of implied to us). Also, you might want to mention that debugging combines very well with "replaying" (and combine with that doc if/when ready) to step through past executions.